e2e/template.go

```diff
@@ -452,7 +452,7 @@ func baseTemplate(location string) *datamodel.NodeBootstrappingConfiguration {
 func getHTTPServerTemplate(podName, nodeName string, isAirgap bool) string {
 	image := "mcr.microsoft.com/cbl-mariner/busybox:2.0"
 	if isAirgap {
-		image = "aksvhdtestcr.azurecr.io/aks/cbl-mariner/busybox:2.0"
+		image = "privateacre2e.azurecr.io/aks/cbl-mariner/busybox:2.0"
```

Comment:
could we use the same var in e2e/scenario_test.go ? var PrivateACRName string = "privateacre2e"

Reply:
fun fact, turns out that global vars defined in _test.go files are not accessible since they are created at runtime. So I'm moving it to a different file so that it can be accessible for all :)